docs/windows-core-installation.md

```diff
@@ -151,7 +169,9 @@ In the **configs** folder you should find:
 
 ```
 authserver.conf.dist
+authserver.conf
 worldserver.conf.dist
+worldserver.conf
```

Comment:
Wrong. The .conf files arent created at build

Reply:
Well more like not needed to run the server. It comes later in the guide. Adding it here could just cause confusion

Reply:
No, with the install step it automatically creates the conf files and not just the dist files, that's part of how your install script works in cmake 😄